### ISSUE TITLE
Adding subfolder tags will add parent tag as well

### ISSUE THREAD

**Issue by christianhans** [sn-extensions/folders-component#17]
Let's say I have a tag #parent.sub contained as subfolder in #parent.

When I use the autocomplete component to assign the tag #parent.sub to my note it will also assign the tag #parent to my note.

I can see why some people would want to do that, but I personally always remove the added parent tag. I guess by adding parent tags someone would be able to find all notes in subfolders. This however gets messy when parent tags are accidentally removed from notes. I rather use Smart Tags to list all notes within a specific folder.

I suggest to change this behavior and not add parent tags when assigning a sub tag to a note.

**Comment by atmoio** [comment]
If we don't do this, and you add subtag `foo.bar`, then when you select `foo` in the folders list, the note with `foo.bar` wouldn't be listed. Super confusing when you have this happen to you ;)

**Comment by christianhans** [comment]
But what happens when someone accidentally deletes foo.bar after it has been added? In that case I won't be able to find that note when selecting foo. I find the current behavior very confusing. At least I would like to be able to disable it via configuration.

**Comment by atmoio** [comment]
If you delete foo.bar, the note will still be in foo, since it was added to both.

**Comment by SiGamma** [comment]
Just adding onto this, I agree with you @mobitar , but right now it's a bit inconsistent, because if I'm in foo.bar and create a new note with the + button, it automatically only tags it with foo.bar, not both. I guess it's expected behavior for users that don't use Folders, though.

**Comment by atmoio** [comment]
Yeah, true, folders functionality needs to be more ingrained within the core, non-extension experience. This will be a work in progress.

**Comment by tab0r** [comment]
I like this behavior, and agree with SiGamma. Make it consistent! :)
